internal/db/schema/migrations/oss/postgres/20/08_plugin_host_views.up.sql

```diff
@@ -6,12 +6,11 @@ alter table host_plugin_host
 create trigger update_version_column after update on host_plugin_host
   for each row execute procedure update_version_column();
 
-drop view host_plugin_host_with_value_obj_and_set_memberships;
-
 -- host_plugin_host_with_value_obj_and_set_memberships is useful for reading a
 -- plugin host with its associated value objects (ip addresses, dns names) and
 -- set membership as columns with delimited values. The delimiter depends on
 -- the value objects (e.g. if they need ordering).
+drop view if exists host_plugin_host_with_value_obj_and_set_memberships;
```

Comment:
shouldn't we know for sure if the view exists or not at this point?

Reply:
Yes, we do know it exists, but this is just extra safety.

Reply:
Just a nit, so I'll go ahead and resolve the conversation after this follow up.

If we know it should exist but it doesn't then that should be a problem we know about.  Tim or Mike (I forget which) suggested I stop using "create or replace function" for this purpose when I know the function wasn't defined previously.

Reply:
I agree with @talanknight. If the view does not exist it is an error and this migration should fail.